odm2api/ODM2/services/readService.py

```diff
@@ -373,16 +373,34 @@ def getProcessingLevels(self, ids=None, codes=None):
 
     # Sampling Feature
     def getSamplingFeatures(self, ids=None, codes=None, uuids=None, type=None, wkt=None, results=False):
-        """
-        * Pass nothing - returns a list of all sampling feature objects with
-          each object of type specific to that sampling feature
-        * Pass a list of SamplingFeatureID - returns a single sampling feature object for the given ids
-        * Pass a list of SamplingFeatureCode - returns a single sampling feature object for the given code
-        * Pass a list of SamplingFeatureUUID - returns a single sampling feature object for the given UUID's
-        * Pass a SamplingFeatureType - returns a list of sampling feature objects of the type passed in
-        * Pass a SamplingFeature Well Known Text - return a list of sampling feature objects
-        * Pass whether or not you want to return only the sampling features
-          that have results associated with them
+        """Retrieve a list of Sampling Feature objects.
```

Comment:
Nice one @lsetiawan! Just rebased from your PR. Can you take a look to see if everything is here?

Reply:
Looks all good. Thanks!